tests/smoke.rs

```diff
@@ -3,13 +3,13 @@
 // SPDX-License-Identifier: Apache-2.0
 //
 
-#[cfg(feature = "fusedev")]
+#[cfg(all(feature = "fusedev", not(feature = "virtiofs")))]
 #[macro_use]
 extern crate log;
 
 mod example;
 
-#[cfg(feature = "fusedev")]
+#[cfg(all(feature = "fusedev", not(feature = "virtiofs")))]
```

Comment:
why disable it for virtiofs?

Reply:
The example tests are only for `fusedev` and fails to compile when `virtiofs` is enabled.